datacontract/export/dbt_converter.py

```diff
@@ -72,9 +72,14 @@ def to_dbt_sources_yaml(data_contract_spec: DataContractSpecification, server: s
     adapter_type = None
     if found_server is not None:
         adapter_type = found_server.type
-        source["database"] = found_server.database
-        source["schema"] = found_server.schema_
+        if adapter_type == "bigquery":
```

Comment:
The specifications for the BigQuery server type are described below.

- https://github.com/datacontract/datacontract-specification?tab=readme-ov-file#bigquery-server-object